src/include/security/levels.rb

```diff
@@ -115,7 +115,7 @@ def initialize_security_levels(include_target)
             "FAIL_DELAY"                                => "6",
             "GID_MAX"                                   => "60000",
             "GID_MIN"                                   => "1000",
-            "DISPLAYMANAGER_SHUTDOWN"                   => "root",
+            "AllowShutdown"                             => "Root",
```

Comment:
ok, not much intuitive. Looks like all desktop developers enjoy creating another layers of abstraction with own names.